### PR TITLE
Implement metric assignment result redesign (part 1)

### PR DESCRIPTION
**Pull request by jessie-ross**
**This PR implements part 1 of the redesign from https://github.com/Automattic/abacus/issues/524:**
- New cards layout with added summary card
- Moved and changed small print

Screenshot in slack.

**Future PRs:**
- Observations card
- Decision explanation
- Altering the outside table to include attr window

## How has this been tested?

- Automated tests that cover added/changed functionality
- Manual testing with production data (locally)

### CODE REVIEW

File: src/components/experiments/single-view/results/MetricAssignmentResults.tsx [hunk 1]
```diff
@@ -227,41 +245,94 @@ export default function MetricAssignmentResults({
...
+                          />
+                          .
+                        </>
+                      }
                     >
-                      Difference
-                    </TableCell>
-                    <TableCell className={classes.monospace}>
+                      <span className={classes.tooltipped}>
+                        <MetricValue
```
Comment (yanirs) [inline src/components/experiments/single-view/results/MetricAssignmentResults.tsx:314]: Per the design, the display of ranges in this table should match that of the top-level display, i.e., show the unit once. It is a bit odd to see something like `-0.21 to +0.2 USD` in the top-level display and `USD -0.21 to USD 0.2` in the details.

To aid with consistency, how about we add `MetricValueInterval` as a component?

**Comment by jessie-ross** [comment]
@yanirs Hmm, actually the 'conversion' interval won't work for revenue metrics, what about "Metric value interval" or changing it to "ARPU interval" just for revenue 🤔

**Comment by yanirs** [comment]
> @yanirs @aaronyan Hmm, actually the 'conversion' interval won't work for revenue metrics, what about "Metric value interval" or changing it to "ARPU interval" just for revenue

Good point. @aaronyan said in the original post:

> * Conversion metrics will have “conversions”, “conversion rate”, and “conversion rate intervals”.
> * Revenue metrics will have “revenue”, “ARPU”, and “ARPU intervals”.

So "ARPU interval" would work. Since we have the space, we can spell it out as "Average revenue per user interval" or "Revenue per user interval". Alternatively, it can be explained in a tooltip.

**Review by aaronyan — APPROVED** [review]
LGTM!

I'm not the biggest fan of the text between the analysis stats card and the graphs

> 95% Credible Intervals (CIs). Experimenter-set minimum practical difference: x.xx pp. Participants: n (n_1 in control; n_2 in treatment).

Since the summary card explanations (min diff amount), observational data card (participants), and graph legends (min diff in context) should have this information, I think we can remove this line in future diffs. :)

If we like having a header to describe the graphs, maybe we have something like?

> Historical 95% credible intervals

I don't have a strong opinion on the naming though. I think my preference is for there to not have any header, since we already have titles for the graphs.

**Comment by jessie-ross** [comment]
> Since the summary card explanations (min diff amount), observational data card (participants), and graph legends (min diff in context) should have this information, I think we can remove this line in future diffs. :)

Yep thats what I was thinking too :)

> I don't have a strong opinion on the naming though. I think my preference is for there to not have any header, since we already have titles for the graphs.

I don't think we need a title for the graphs but I think we will need to mention somewhere that the analyses are 95% intervals.

**Pull request merged**